### ISSUE TITLE
Support create_user

### ISSUE THREAD

**Issue by PlasmaSheep** [kvesteri/sqlalchemy-utils#240]
Since we already have `create_database`, it would be nice to have `create_user` for database initialization purposes.

**Comment by spyoungtech** [comment]
I agree. I am wondering if this is possible to do with the exiting `quote` functions.
